twig/author_layout/simple_list.twig

```diff
@@ -6,7 +6,7 @@
     <ul>
         {% for author in authors %}
             <li class="author_index_{{ loop.index }}">
-                <a href="{{ author.link }}" class="{{ item_class }}" rel="author">{{ author.display_name }}</a> - {{ author.age }}
+                <a href="{{ author.link }}" class="{{ item_class }}" rel="author">{{ author.display_name }}</a> {{ author.age }}
```

Comment:
@htmgarcia please, could you remove the `{{ author.age }}` variable as well? It is there by mistake, a leftover from an old test.

Reply:
Ah, never mind. We can merge your changes and I remove that variable.